encoding/gocode/gen.go

```diff
@@ -42,10 +42,10 @@ type TypeConfigOpenAPI struct {
 	// lowercase version of the Lineage.Name() is used.
 	PackageName string
 
-	// ApplyFuncs is a slice of AST manipulation funcs that will be executed against
+	// ApplyFuncs is a slice of DST manipulation funcs that will be executed against
```

Comment:
```suggestion
	// ApplyFuncs is a slice of AST manipulation funcs that will be executed against
```

i think best to leave this one as "AST", as this is referring to the general concept of abstract syntax trees, not the particular implementation.